public/app/ui/components/collections/library/list-result.html

```diff
@@ -1,11 +1,11 @@
-<div class="list-result list-result-library">
-    <span class="pull-left" data-bind="component: displayFor('MediaThumbnail', result)"></span>
-    <span class="pull-right label label-default" data-bind="component: displayFor('type', result)"></span>
-    <a class="result-title" href="#" data-bind="attr: { href: detailUrlFor(result) }, component: displayFor('Title', result)"></a>
+<a href="#" data-bind="attr: { href: detailUrlFor(result) }" class="list-result list-result-library clearfix">
```

Comment:
The `*-result` views now have the `<a>` as the top-level container, so that the whole thing is clickable.  Applied this to both `list` (as here) and `thumbnails` views.